cs/libdpr/samples/DprCounters/DprCounters/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/aspnet:6.0
```

Comment:
This is not generated. This part of the process cannot be generated and has to look like this since it's codebase specific.  This file is implicitly used when building an image of the codebase with the docker_stuff script